compiler/cpp/src/thrift/generate/t_rs_generator.cc

```diff
@@ -1141,13 +1141,18 @@ void t_rs_generator::render_struct_impl(
     render_struct_constructor(struct_name, tstruct, struct_type);
   }
 
-  render_struct_sync_read(struct_name, tstruct, struct_type);
-  render_struct_sync_write(tstruct, struct_type);
-
   if (struct_type == t_rs_generator::T_RESULT) {
     render_result_struct_to_result_method(tstruct);
   }
 
+  f_gen_ << "}" << endl;
```

Comment:
Please fix indentation here. You need to add an `indent_down()` call before closing the `impl` block..